src/common/snippets/tests/include/pass/mha_tokenization.hpp

```diff
@@ -17,7 +17,7 @@ class TokenizeMHASnippetsTests : public TransformationTestsF {
     virtual void run();
 
 protected:
-    ov::snippets::pass::SnippetsTokenization::Config config;
+    ov::snippets::pass::SnippetsTokenization::Config config { 1, std::numeric_limits<size_t>::max(), true, true, { 3, 4 } };
```

Comment:
I see that you manually initialize default config in each test. I think we can create default config for all unit snippets tests. What do you think?
For example, create the common separate header with one function `static Config get_default_config() { return { 1, std::numeric_limits<size_t>::max(), true, true, { 3, 4 } }; }`
Another moment, do we really need to init each param of config in tests?

Reply:
Thanks Alexandra for the comment! This revision is made according to our disscusion chain here https://github.com/openvinotoolkit/openvino/pull/23180#discussion_r1540425533. We want to avoid plugin user use the default constructor of ov::snippets::pass::SnippetsTokenization::Config. So config from test side also must be inited explicitly. I applied the static function you suggested in class SnippetsTokenization, as such config is needed in common snippets tests as well as plugin snippets tests. So we can avoid manually initialization of config, and still avoid plugin user use default constructor unintentially.

Reply:
Yeah, I got that you had to do so because of https://github.com/openvinotoolkit/openvino/pull/23180#discussion_r1540425533. But you always initialized this config with the same parameters. So I suggested to create common static getter.
But I'm not sure that to add this method to `Tokenization` class is good idea. Because `default` meaning can be interpreted differently by each backend. So I suggested to create this method in tests, probably in the separate file since it's used ONLY in tests. Could you take a look at this moment? Is it possible to do in test infrastucture?

Reply:
Applied. Thanks Alexandra!